tests/integration/README.md

```diff
@@ -0,0 +1,33 @@
+# Integration tests with requre support
```

Comment:
Question/brainstorm: I see the content is a bit longer, but still, wouldn't it make sense to store this detailed information about the test under `description` in the `main.fmf`? Now `tmt test show integration` gives just two brief summaries.

Reply:
seems like good idea. I can add it into description instead of `README`, second position is, that README is automatically visible on github in the repo page.